<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepEdit</class>
 <widget class="QDialog" name="StepEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_left">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>0</y>
     <width>1166</width>
     <height>768</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_17">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>3</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>步骤配置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>407</x>
       <y>92</y>
       <width>721</width>
       <height>571</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
    font: 14pt &quot;黑体&quot;; 
	border-width:1px;  
	border-style:solid;
	border-color: rgb(51, 153, 255);  
	background-color: rgb(248, 248, 255);
}
QTableWidget::Item{

}
QTableWidget::Item:hover{

}
QTableWidget::item:selected{
	background:lightgray; 
	color:rgb(248, 248, 255); 
}
QTableWidget::item:selected:!active{

}
QTableCornerButton::section{
	background-color: rgb(51, 153, 255);
}
QScrollBar:vertical{
width:36px;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="pushButton_insert">
     <property name="geometry">
      <rect>
       <x>532</x>
       <y>682</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_remove">
     <property name="geometry">
      <rect>
       <x>726</x>
       <y>682</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_append">
     <property name="geometry">
      <rect>
       <x>431</x>
       <y>682</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>407</x>
       <y>92</y>
       <width>50</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 153, 255);
color: rgb(248, 248, 255);
font:14pt</string>
     </property>
     <property name="text">
      <string>序号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_plcDB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>214</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_plcDB">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>216</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>PLC的DB站号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_plcIP">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>304</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>PLC的IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_plcIP">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>302</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_auto">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>118</y>
       <width>125</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
     </property>
     <property name="text">
      <string>自动工位：</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_manual">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>118</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
     </property>
     <property name="text">
      <string>手动工位：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1049</x>
       <y>680</y>
       <width>54</width>
       <height>54</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/saves.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_update">
     <property name="geometry">
      <rect>
       <x>631</x>
       <y>682</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_warning">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(250, 0, 0);</string>
     </property>
     <property name="text">
      <string>请填写完整!</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
background-color: rgb(51, 153, 255);




</string>
   </property>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>40</y>
      <width>57</width>
      <height>57</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: url(:/re/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rfid">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>167</y>
      <width>17</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/triangle.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rfid">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
    background:transparent;
    font: 18pt &quot;黑体&quot;;
    color: rgb(248, 248, 255);
}</string>
    </property>
    <property name="text">
     <string>步骤配置</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_black">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>760</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
